library/json_stat.cpp

```diff
@@ -40,6 +40,14 @@ static void eblob_stat_global_json(struct eblob_backend *b, rapidjson::Value &st
 {
 	for (uint32_t i = EBLOB_GST_MIN + 1; i < EBLOB_GST_MAX; i++)
 		stat.AddMember(rapidjson::StringRef(eblob_stat_get_name(b->stat, i)), eblob_stat_get(b->stat, i), allocator);
+
+	stat.AddMember(rapidjson::StringRef("datasort_total_data_size"),
```

Comment:
`rapidjson::StringRef()` is unnecessary here

Reply:
done